src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
I am not sure...  In the past I have kept code with optional dependencies in sub packages, as a `best practice`, due to other tooling commonly found in Java tool chains.  This may no longer be a concern.